CONTRIBUTING.md

```diff
@@ -335,11 +335,18 @@ Mocha tests are contained in `__tests__` directories.
 #### Jest
 Jest tests are stored in the same directory as source code files with the `.test.js` suffix.
 
-### Running Jest Unit Tests
-
-```bash
-node scripts/jest
-```
+### Running Kibana tests
+
+The following table outlines possible test file locations and how to invoke them:
+
+| Test location                                                                                                                                           | Runner command (working directory is kibana root)                                       |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
+| `src/**/*.test.js`<br>`src/**/*.test.ts`                                                                                                                | `yarn test:jest -t=regexp [test path]`                                                  |
+| `src/**/__tests__/**/*.js`<br>`packages/kbn-datemath/test/**/*.js`<br>`packages/kbn-dev-utils/src/**/__tests__/**/*.js`<br>`tasks/**/__tests__/**/*.js` | `yarn test:mocha --grep=regexp [test path]`                                             |
```

Comment:
`yarn test:mocha --grep=kbn-datemath` doesn't work for me, runs all tests anyway. what did I wrong?

Reply:
I assumed `yarn test:mocha --grep='multiple values' ./x-pack/server/lib/__tests__/kibana_state.js` would be the same as `--grep='multiple values' ./x-pack/server/lib/__tests__/kibana_state.js` but seems like it's not, I'll fix the command.